avocado/core/varianter.py

```diff
@@ -467,6 +467,15 @@ def to_str(self, summary=0, variants=0, **kwargs):
         :param kwargs: Other free-form arguments
         :rtype: str
         """
+        if self._no_variants == 0:  # No variants, only defaults:
+            out = []
+            if summary:
+                out.append("No variants available, using defaults only")
+            if variants:
+                out.append("\n".join(variant_to_str(next(self.itertests()),
+                                                    variants - 1, kwargs,
+                                                    self.debug)))
+            return "\n\n".join(out)
```

Comment:
`None` is python representation of absence of value. Human equivalent representation output should be indeed `""`. If you want to expose internals, the path should then be `/run`:
```
dict: {'mux_path': '/run', 'variant': [TreeNode(name='')], 'variant_id': None}
```
..but I don't think we should go that way.